### PR TITLE
Update picture-in-picture idlharness test

### PR DESCRIPTION
**Pull request by stephenmcgruer**
The actual idl update already happened in
https://github.com/web-platform-tests/wpt/commit/1449b69c394008d51cb625bd1adbde782c01e8be,
but the associated test was not updated.

Closes https://github.com/web-platform-tests/wpt/pull/24226

### CODE REVIEW

**Comment by stephenmcgruer** [comment]
We may wish to hold off on landing this until the Chromium change (landed today) makes it to dev and we can verify this update.

**Review by mounirlamouri — APPROVED** [review]

**Pull request merged**
